### ISSUE TITLE
Css to fix on posts page on smarthpone

### ISSUE THREAD

**Issue by bebatut** [open-life-science/open-life-science.github.io#238]
![Screenshot_20201207-125123.png](https://user-images.githubusercontent.com/1842467/101347859-fa3ee400-388a-11eb-81cb-a6ed4f6d7834.png)

**Comment by bebatut** [comment]
![Screenshot_20201207-125226.png](https://user-images.githubusercontent.com/1842467/101347937-1b073980-388b-11eb-97f0-7536fd55692d.png)

**Comment by NPDebs** [comment]
Hi @bebatut 👋🏼 
Has this issue been fixed? 👀 

If no, I'd like to take this on.

**Comment by kryptosjay** [comment]
Hi am jacinta 
I am an outreachy applicant and would like to work on this issue.
if given access please

**Comment by Jolah1** [comment]
Hello @bebatut 
My name is Jolade, I am an Outreachy applicant, I will be working on this issue.
thank you.

**Comment by bebatut** [comment]
As defined in the [OLS Outreachy contribution guide](https://docs.google.com/document/d/1HP7qTBe-6YrCUYAVkrT441rGtF6zxzNbbz2E5RWQvTM/edit?usp=sharing), 
- If someone else has already commented, let them try first
- If they don’t update after three days, the issue is “free” again, and you can pick it up. 

@NPDebs were the first one to comment here. Are you still interested to work on this issue? You are already interested in other issues. You might need to pick one

**Comment by Jolah1** [comment]
Thanks for replying ma’am, it’s seems most issues have been taken up at the
moment…..I would really appreciate if you can assign anyone that is free to
me please.

This will be my first contribution

Thank you
Best Regards.

**Comment by bebatut** [comment]
I am sorting issues and will add more in the next hours

**Comment by Jolah1** [comment]
Alright ma'am

**Comment by NPDebs** [comment]
> As defined in the [OLS Outreachy contribution guide](https://docs.google.com/document/d/1HP7qTBe-6YrCUYAVkrT441rGtF6zxzNbbz2E5RWQvTM/edit?usp=sharing),
> 
> * If someone else has already commented, let them try first
> * If they don’t update after three days, the issue is “free” again, and you can pick it up.
> 
> @NPDebs were the first one to comment here. Are you still interested to work on this issue? You are already interested in other issues. You might need to pick one

Hi @bebatut 
These blocks of code (added to the custom.css file under the _sites directory) caused the required behaviour when I tested the site in my gitpod environment. However, git ignores this folder and so, the changes cannot be committed. I know I'm doing something wrong. Please help in any way you can. Thank you!

```
@media screen and (min-width: 320px) and (max-width: 720px), print {
  .hero {
    background-attachment: fixed;
    height: auto;
    min-height: 400px;
  }
  .hero .hero-body h1 {
    font-size: 2.5em;
    line-height: 1.2em;
    margin-bottom: 50px;
  }
}
```
![ols-pr (2)](https://user-images.githubusercontent.com/105166953/195134038-905d55c5-0a24-429b-a577-c6e0afa0872a.png)

**Comment by NPDebs** [comment]
Hello again, @bebatut 😅

While reviewing someone else's PR, I think I got the hang of where the styles should go.

Please review the changes to see if everything is in order. Thank you! 🙌🏼

**Comment by bebatut** [comment]
@NPDebs seems to have fixed the issue here #449
Does anyone want to review the Pull Request, test it locally and check if the issue is really fixed?

**Comment by NPDebs** [comment]
Hi, @bebatut...I think you may have forgotten to close this issue. 👀
